library/src/main/scala/scala/scalajs/runtime/EnvironmentInfo.scala

```diff
@@ -49,4 +49,20 @@ trait EnvironmentInfo extends js.Object {
   def sourceMapper: js.UndefOr[js.Function1[
     js.Array[JSStackTraceElem], js.Array[JSStackTraceElem]]] = js.native
 
+  /** Semantics configuration.
+   *
+   *  @group envInfo
+   */
+  val semantics: EnvironmentInfo.Semantics = js.native
```

Comment:
Who needs this change? Maybe we should hold this off for a while to try and not change the IR too often...